tests/RegistryPageFunctionalTest.php

```diff
@@ -6,22 +6,46 @@
 use SilverStripe\Dev\CSSContentParser;
 use SilverStripe\Dev\FunctionalTest;
 use SilverStripe\Registry\Tests\Stub\RegistryPageTestContact;
+use SilverStripe\Registry\Tests\Stub\RegistryPageTestContactExtra;
 use SilverStripe\Registry\Tests\Stub\RegistryPageTestPage;
 
 class RegistryPageFunctionalTest extends FunctionalTest
 {
     protected static $fixture_file = [
-        'fixtures/RegistryPageTestContact.yml',
         'fixtures/RegistryPageFunctionalTest.yml',
+        'fixtures/RegistryPageTestContact.yml',
+        'fixtures/RegistryPageTestContactExtra.yml'
     ];
 
     protected static $extra_dataobjects = [
         RegistryPageTestContact::class,
-        RegistryPageTestPage::class,
+        RegistryPageTestContactExtra::class,
+        RegistryPageTestPage::class
     ];
 
     protected static $use_draft_site = true;
 
+    public function testUseLink()
+    {
+        // Page with links
+        $page = $this->objFromFixture(RegistryPageTestPage::class, 'contact-registrypage-extra');
+        $response = $this->get($page->Link());
+        $parser = new CSSContentParser($response->getBody());
+
+        $rows = $parser->getBySelector('table.results tbody tr td');
+
+        $this->assertEquals($rows[0]->a->attributes()->href[0], '/contact-search-extra/1');
```

Comment:
The expected value should be the first argument, with the result being the second argument. Also this kind of assertion has a tendency to break when users have different themes installed, or have a different base path etc. Perhaps you could switch it for `$this->assertContains('/contact-search-extra/1', $rows[.....);` as a partial match instead?